### PR TITLE
[camera_android] Add NV21 as an image stream format #3277

### PR DESCRIPTION
**Pull request by acoutts**
This contains the changes for camera_android from https://github.com/flutter/packages/pull/3277

### CODE REVIEW

**Review by bparrishMines — APPROVED** [review]
LGTM with merge from main

**Comment by bparrishMines** [comment]
cc @camsim99 for secondary review. This is the android implementation from https://github.com/flutter/packages/pull/3277

File: packages/camera/camera_android/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## 0.10.5
 
+* Adds support for NV21 as a new streaming format in Android which includes correct handling of image padding when present.
+
```
Comment (camsim99) [inline packages/camera/camera_android/CHANGELOG.md:4]: Nit: remove extra space

**Review by camsim99 — APPROVED** [review]
LGTM! Just left nit about spacing

File: packages/camera/camera_android/CHANGELOG.md (hunk 1 above)
Comment (acoutts) [reply packages/camera/camera_android/CHANGELOG.md]: Fixed in 76dd696960fb6f35467e0cc763afec425cde5de1

Comment (acoutts) [reply packages/camera/camera_android/CHANGELOG.md]: Fixed in 76dd696960fb6f35467e0cc763afec425cde5de1

**Comment by bparrishMines** [comment]
We recently added enforcement of lint warnings for Android. This PR is currently waiting on https://github.com/flutter/packages/pull/3716

**Comment by acoutts-nydig** [comment]
Just merged with main, seeing if things light up green next.

**Comment by stuartmorgan** [comment]
> Lint found 16 errors

You can either run the `lint-android` repo tooling command locally and open the HTML file that the logging references, or you can [turn on all the Android lints in AS](https://developer.android.com/studio/write/lint#cis) to see these locally.

**Comment by acoutts-nydig** [comment]
Got it - just pushed a fix for those, says no issues locally now.

**Pull request merged**